src/index.ts

```diff
@@ -12,7 +12,9 @@ export function createApp(targetDomain?: string) {
     domain = url.searchParams.get("domain") || "";
   }
 
-  app.setState({ domain });
+  id = new URL(domain).searchParams.get("id") ?? "";
```

Comment:
Nitpick but it's not changed anywhere

```suggestion
  const id = new URL(domain).searchParams.get("id") ?? "";
```